examples/store-mongodb-similarity-search.php

```diff
@@ -55,7 +56,7 @@
 
 // create embeddings for documents
 $platform = new OpenAI(HttpClient::create(), $_ENV['OPENAI_API_KEY']);
-$embedder = new DocumentEmbedder($embeddings = new Embeddings($platform), $store);
+$embedder = new DocumentEmbedder($embeddings = new Embeddings($platform), $store, new NullLogger());
```

Comment:
`NullLogger` is fine here?

Reply:
could also be part of the constructor as default, but don't really have a preference tbh

Reply:
will do

Reply:
done